composer.json

```diff
@@ -55,7 +55,12 @@
         }
     },
     "scripts": {
-        "test": "./vendor/bin/pest --parallel && ./vendor/bin/phpstan analyse"
+        "test:pest": "pest --parallel",
```

Comment:
Don't we still need the `./vendor/bin` prefix though?

Reply:
Composer is witty: puts `./vendor/bin` in the beginning of `$PATH`
https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands (purple callout)
Even on Windows!

Reply:
Awesome, TIL!